i18n/en.json

```diff
@@ -1332,7 +1332,7 @@
   "admin.team.openServerTitle": "Enable Open Server: ",
   "admin.team.restrict_direct_message_any": "Any user on the Mattermost server",
   "admin.team.restrict_direct_message_team": "Any member of the team",
-  "admin.team.restrictDescription": "Teams and user accounts can only be created from a specific domain (e.g. \"mattermost.org\") or list of comma-separated domains (e.g. \"corp.mattermost.com, mattermost.org\").",
+  "admin.team.restrictDescription": "Teams and user accounts can only be created from a specific domain (e.g. \"mattermost.org\") or list of comma-separated domains (e.g. \"corp.mattermost.com, mattermost.org\"). This setting only affects email login.",
```

Comment:
Should it be "Teams and user accounts" or only "User accounts"? In which case does it affect Team creation?

Reply:
@DSchalla Good catch. This is a remnant of a (very) old system when team and user accounts could be restricted based on domain.

Updated to only say "User accounts".